public/js/loginValidation.js

```diff
@@ -0,0 +1,50 @@
+let isValidEmail = (elementID) => {
+    let inputVal = document.getElementById(elementID)
+    let userEmailWarning = document.getElementById('userEmailWarning')
+    let email = inputVal.value.trim();
+    let regEx = new RegExp("^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$")
+    if (regEx.test(email)) {
+        userEmailWarning.classList.remove('error')
+        return true
+    } else {
+        userEmailWarning.classList.add('error')
+        return false
+    }
+}
+
+let validateInputs = (cleanedEmailInput, cleanedPasswordInput) => {
+    if (isValidEmail('userEmail')) {
```

Comment:
uv already got the email string here, why are u re-getting it from the DOM? :/

Reply:
done.